### ISSUE TITLE
Clarity website renders application header icon oddly

### ISSUE THREAD

**Issue by joshjohnsonux** [vmware/clarity#5393]
## Describe the bug

In reviewing the new Clarity website, it appears that the icons in the App Header component aren't rendering correctly.

1.  Go to Clarity's new website (https://next.clarity.design/)
2.  Click on Navigation
3.  Scroll down to examples that include the App Header component
4.  See error in screenshot
![Screen Shot 2020-12-17 at 12 50 00 PM](https://user-images.githubusercontent.com/8105536/102542968-71346380-4067-11eb-8acb-181a1b082c76.png)

## Versions

**Device:**

- Type: Macbook
- OS: MacOS 10.15.7
- Browser: Chrome
- Version: 87.0.4280.88

**Comment by joshjohnsonux** [comment]
Note that the issue doesn't exist on the old website.
![Screen Shot 2020-12-17 at 1 03 15 PM](https://user-images.githubusercontent.com/8105536/102543584-5ca49b00-4068-11eb-95b2-366310731a26.png)

**Comment by bdryanovski** [comment]
Hey @joshjohnsonux, thanks for reporting this, I did not manage to reproduce the issue

<img width="914" alt="Screenshot 2021-01-05 at 12 31 50" src="https://user-images.githubusercontent.com/204564/103636044-17caa080-4f52-11eb-8437-5f1a80e996df.png">

So it could be fixed from one of the previous fixes that we made or it could be some local glitch at some point.

If someone could provide reproducible steps please reopen the issue.

**Comment by joshjohnsonux** [comment]
Odd. I am consistently reproducing it on my systems.  I've cleared cache and gone incognito with no extensions, and that hasn't helped.  I tried using latest Firefox instead to isolate it as a Chrome issue, but the problem exists there as well.  I also tried both on a Mac, a Windows machine, and even a clean Windows VM.  Issue appeared consistently across them.  Only thought left is that the image file is being blocked at the network level by either ISP or router, though I can't really go elsewhere to test due to Covid.  Would be useful to have someone on a 3rd network check what they're seeing.
